src/main/java/chess/domain/piece/EmptyPiece.java

```diff
@@ -0,0 +1,29 @@
+package chess.domain.piece;
+
+import chess.domain.position.Direction;
+import chess.domain.position.Position;
+import java.util.List;
+import java.util.Map;
+
+public class EmptyPiece extends Piece{
```

Comment:
```suggestion
public class EmptyPiece extends Piece {
```

포맷팅 빠진 부분이 없는지 점검해 보아요.

Reply:
포맷팅 신경쓰도록 하겠습니다 :(